analyzers/tests/SonarAnalyzer.Test/LiveVariableAnalysis/RoslynLiveVariableAnalysisTest.TryCatchFinally.cs

```diff
@@ -65,7 +65,7 @@ public void Using_LiveInUntilTheEnd(string usingStatement, string suffix)
     [TestMethod]
     public void Using_Nested_Block_LiveInUntilTheEnd()
     {
-        var code = """
+        const string code = """
```

Comment:
`cons string` has no performance benefit over `var`:
https://stackoverflow.com/a/49224799
This creates unnecessary visual noise.